src/test/java/Utils/GenericUtils.java

```diff
@@ -0,0 +1,19 @@
+package Utils;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+import java.util.ArrayList;
+
+public class GenericUtils {
+    public WebDriver driver;
+
+    public GenericUtils(WebDriver driver){
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    public void SwitchToNewTab(){
+        ArrayList<String> tabs = new ArrayList<String>(driver.getWindowHandles());
+        driver.switchTo().window(tabs.get(1));
```

Comment:
Deze is een beetje misleidend. Volgens mij opent deze functie niet de nieuwe tab als er al eerder een extra tab geopend was. Als je deze functionaliteit naar een utilities class verhuisd zou je er ook voor willen zorgen dat deze werkt ongeacht in welke situatie je deze toe gaat passen.
`driver.switchTo().window(tabs.get(tabs.size() - 1));`
Dit zou echt altijd moeten werken.